docs/manual/lore/faction-proteron.md

```diff
@@ -0,0 +1,61 @@
+
+As time passed, House Proteron began to notice the cracks in the old Empire. Already unhappy with the Emperor's reticence in dealing decisively with the Sirii, the Empire's handling of the Dvaered disgusted the Proteron. Rather than reassigning the workers and improving their living standards to allow them to reach their true potential, the Empire gave up on them. In addition, it had been centuries since the Za'lek had given more than the cursory nod to Imperial authority. The House generally blamed the Empire's social structure for its inability to command the loyalty of its subjects. The House, on the other hand, had grown by leaps and bounds, claiming more and more space for their systems, even bordering Sol. The final straw was the Quarantine of Sorom, where the Empire simply laid down and let a disease claim a whole planet.
+
+Some debate followed, but the consensus was that House Proteron had achieved its original goal, and now it was time for the weak Empire to make way for the new generation of galactic dominance. But when the Proteron High Autarch delivered this message to the Emperor, he was furious at the perceived insult and ordered all Autarchs executed. Appalled at this reaction, the Circle concluded in an emergency meeting that the Empire had no intention of honoring the original agreement, thereby forfeiting its claim to the loyalty of House Proteron. It was clear to the Circle that should the Empire insist on trying to turn the Proteron into another Dvaered incident or, worse still, turning them into Empire systems, there could be no peace with them.
+
+Knowing that they could not at the time resist the entire might of the Empire's infrastructure, the Proteron decided to plant their agents in every major Imperial agency while publicly grovelling at the Empire's feet. Soon enough, the Proteron were even in charge of the Empire's covert operations against the Proteron. Now having loosened their leash greatly, they set out to prepare for a galaxy-scale war. While they worked, the shape of the galaxy was changing, so to speak.
+
+New advances in quantum engineering led to the development of quantum transtators and translators, alternative travel engines capable of bypassing warp points and so changing conventional military doctrine, which recommended fortifying chokepoint systems on the warp network. Expecting correctly that the Empire had not even thought of this change, the Proteron decided to bypass their defences using the new hypergate being built in Sol. After all, once the Emperor and their successors were in their hands, the war would be essentially meaningless. They built a hypergate in secret in their home system, Protera. They seceded from Imperial authority as soon as the gate was built and moved a small fleet as a feint into the classical defences. When the Empire took the bait, the Sovereign Proteron Autarchy, as they now styled themselves, sent the most massive fleet ever seen through their hypergate.
```

Comment:
I would avoid mentioning quantum stuff, I don't think it's clear yet what makes the hypergates work.

Reply:
Oh ok. I thought that since the construction heads of the various factions say that it's powered by some quantum translator stuff which was also there in Sol, the QT was canon.

Reply:
That was a bit of fumbling by my part, but they say "It's some sort of high power quantum energy translator or something like that.", so it's not necessarily set in stone, they could just be getting confused :D

Reply:
True, but the hypergate is only implied to be powered by the QT here. I could remove all mentions of it, but it's not explicit here

Reply:
Well, I guess it could be kept pending potential changes in the future. I'll resolve this then.